packages/edit-post/src/components/layout/index.js

```diff
@@ -145,7 +145,7 @@ function Layout() {
 									</div>
 								) }
 								<SettingsSidebar />
-								<Sidebar.Slot />
+								<PluginComplementaryArea.Slot scope="edit-post" />
```

Comment:
Can you explain the naming here? `PluginComplementaryArea`?

Reply:
Actually, what I'm missing in this PR is documentation, what are these components for? How to introduce plugins to a new screen? This would help personally understand the APIs and provide better feedback.

Reply:
Hi @youknowriad, I added docs to the API we are exposing.
This PR is exposing two actions, two selectors, and two pairs of slot fill components.
Let me know If the documentation helped to understand how the mechanism we are adding work.

Reply:
That's useful 👍